doc/postman_collection.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I opted for a Postman collection because of his simplicity and because can be immediately used importing it in Postman